config/ftbquests/quests/chapters/bounty_board.snbt

```diff
@@ -531,13 +526,26 @@
 					type: "xp"
 					xp: 1000
 				}
+				{
+					id: "380E221DF1F736A8"
+					type: "random"
+					exclude_from_claim_all: true
+					table_id: 5564196992594175882L
+				}
 			]
 		}
 		{
 			title: "Kill the Trader Villager and His Annoying Llamas"
 			x: 3.0500000000000003d
 			y: 6.0d
 			subtitle: "\"This is Personal\" - AlfredGG"
+			description: [
+				"\"While making a lot of the questlines for the ATM7, the Trader Villager thought it'd be funny to constantly push me around in the quest screen."
```

Comment:
```suggestion
				"\"While making a lot of the questlines for ATM8, the Trader Villager thought it'd be funny to constantly push me around in the quest screen."
```
Alternative, rename "ATM7" to "ATM series"

Reply:
This happened while writing the quests for ATM7.